### DISCUSSION TITLE
looking for a way to have the first time chatter more than once

### DISCUSSION THREAD

**Discussion by miqueet** [Chatterino/chatterino2#4558] [Q&A] [answered]
Hey all

looking for a way to have the first time chatter notification come up once per stream for each person so it makes it easier to identify a new chatter in the stream for that day. This seems doable and would be a great configuration. Is this possible to do with the current software?

**Comment by Felanbird** [comment, marked as answer]
First time chatter is only displayed due to a tag Twitch sends us, if the message is the users first ever message in the channel, Twitch sends us `first-msg=1` otherwise for _every_ other message in the channel, they send us `first-msg=0`.

So we have no way of distinguishing if a message was the users first message of the day.

**Reply by pajlada** [reply]
You could potentially implement your own "first message of the day" logic with plugins - but as for Twitch's feature that's all you can get.
